netscaler/config_test.go

```diff
@@ -115,6 +116,60 @@ func TestAdd(t *testing.T) {
 	}
 }
 
+func TestApply(t *testing.T) {
+
+	rndIP := randomIP()
```

Comment:
AFAIK, Apply only applies to ACLs. Can you rewrite the test to apply ACL?

Reply:
I guess that's what the `ApplyResource` function is doing `apply ns acls`, applying the configured changes for acls. I am not sure if there is any such thing as such `apply ns acl` : http://docs.citrix.com/ko-kr/netscaler/11/reference/netscaler-command-reference/ns/ns-acl.html . Correct me if I am missing to read your question.

Reply:
What I meant was that TestApply as written tests the creation of lbvserver. It should test apply of ACL.